### PR TITLE
[SYCLomatic] Cleanup helper function src files

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/runtime/dpct-rt/CMakeLists.txt [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
+  set( src ${CMAKE_CURRENT_SOURCE_DIR}/${f} )
+  set( dst ${output_dpstd_dir}/${f} )
+  add_custom_command(OUTPUT ${dst}
+    DEPENDS ${src}
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${src} ${dst}
+    COMMENT "Copying clang's ${f}...")
+  list(APPEND out_files ${dst})
+endforeach( f )
+add_custom_target(dpct-headers ALL DEPENDS ${out_files})
+#add_custom_target(dpct-headers ALL DEPENDS dpct_helper_headers_and_inc)
```
Comment (zhiweij1) [inline clang/runtime/dpct-rt/CMakeLists.txt:55]: remove

Comment (zhiweij1) [inline clang/runtime/dpct-rt/CMakeLists.txt]:
```suggestion

```

**Review by ziranzha — APPROVED** [review]

File: clang/runtime/dpct-rt/processFiles.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+def get_files(path):
+    items = os.listdir(path)
+    files = []
+    for item in items:
+        if (os.path.isfile(os.path.join(path, item))):
+            files.append(item)
+    return files
+
+def main():
```
Comment (zhimingwang36) [inline clang/runtime/dpct-rt/processFiles.py:96]: Please further improve the python scripts on readme to include desc and howto.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

File: clang/runtime/dpct-rt/processFiles.py (hunk 2 above)
Comment (zhiweij1) [reply clang/runtime/dpct-rt/processFiles.py:96]: OK, I will update it in a separate PR.

**Pull request merged**